credits.md

```diff
@@ -21,7 +21,7 @@ TiDB developers contribute to new feature development, performance improvement,
 - [pingcap/dm](https://github.com/pingcap/dm/graphs/contributors)
 - [pingcap/tidb-binlog](https://github.com/pingcap/tidb-binlog/graphs/contributors)
 - [pingcap/tidb-dashboard](https://github.com/pingcap/tidb-dashboard/graphs/contributors)
-- [pingcap/ticdc](https://github.com/pingcap/ticdc/graphs/contributors)
+- [pingcap/ticdc](https://github.com/pingcap/tiflow/graphs/contributors)
```

Comment:
`pingcap/br` and `pingcap/tidb-lightning` belong to `pingcap/tidb` now
`pingcap/dm` and `pingcap/ticdc` belong to `pingcap/tiflow` now.
How should we treat these repos?

Reply:
About DM versions, there's a note in this doc. https://docs.pingcap.com/zh/tidb/stable/dm-overview/
About pingcap/br and pingcap/tidb-lightning merged into pingcap/tidb, we'll check if it affects docs.